src/containers/Header/index.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react'
+import { Link } from 'react-router-dom'
+
+import { lightBlue400 } from 'material-ui/styles/colors'
+import AppBar from 'material-ui/AppBar'
+
+import cx from 'classnames/bind'
+import style from './style.scss'
+
+import UtilMenu from '../../components/UtilMenu'
+
+const classNames = cx.bind(style)
+
+class Header extends Component {
+  state = {
+    logged: false
+  }
+
+  onClickSign = () => {
+    this.setState({ logged: !this.state.logged })
+  }
+
+  render () {
+    return (
+      <AppBar
+        title={
+          <Link className={classNames('Header__brand')} to='/'>BaapChef</Link>
+        }
+        iconElementRight={
+          <UtilMenu
+            isLogged={this.state.logged}
+            onClickSign={this.onClickSign} />
+        }
+        style={{
```

Comment:
Can we use the library `material-ui/styles/colors` colors in .scss file? If yes, we should put the style setting to `style.scss`

Reply:
nope, we cannot do that. scss is totally separate file so it cannot be integrated with external library.